kedify-agent/values.yaml

```diff
@@ -38,6 +38,7 @@ agent:
         disabled: false
       kubernetesResources:
         interval: 30
+        disabled: true
```

Comment:
it's actually correct. If it's not there at all, it's `disabled: false` because false is the 0 value for bool, but for this one it should be actually disabled, same as in webui. This flat controls if the metrics for pods and hpas and jobs should be collected in other namespaces than the installation ns.